revenue_app/templates/revenue_app/dashboard.html

```diff
@@ -8,5 +8,164 @@
 
 {% block content %}
 <h1>Dashboard</h1>
-
+<br>
+<div class="container mt-4">
+  <div class="row">
+    <div class="col-6">
+      <h4>Argentina</h4>
+      <table class="table table-striped mt-4">
+        {% for key, value in summarized_data.ars_data.items %}
+          <tr>
+            <td><b>{{key}}</b></td>
+            <td class="text-right">{{value}}</td>
+          </tr>
+        {% endfor %}
+      </table>
+    </div>
+    <div class="col-6">
+      <h4>Brazil</h4>
+      <table class="table table-striped mt-4">
+        {% for key, value in summarized_data.brl_data.items %}
+          <tr>
+            <td><b>{{key}}</b></td>
+            <td class="text-right">{{value}}</td>
+          </tr>
+        {% endfor %}
+      </table>
+    </div>
+  </div>
+</div>
```

Comment:
Could we build this on the BE and iterate here(FE) per country? this will allow us to avoid add one table per every new country, and also, not to hardcode them